gmap-custom-marker.vue

```diff
@@ -176,6 +176,9 @@ export default {
     }
   },
   destroyed() {
+    if (this.$clusterObject) {
+      this.$clusterObject.removeMarker(this.$overlay, true);
+    }
     this.$overlay.setMap(null);
     this.$overlay = undefined;
```

Comment:
```suggestion
       if (this.$clusterObject) {
      this.$clusterObject.removeMarker(this.$overlay, true);
    } else {
      this.$overlay.setMap(null);
      this.$overlay = undefined;
    }
```

Reply:
I merged too quickly. I integrate your code by hand Aminerman.

Reply:
@Evertt It does work this way and didn't notice any bug. But can you describe your case?
I'll publish a demo, before the end of the week, that uses vue-google-maps with this custom marker and the cluster. That way we'll have a common test ground.